samples/snippets/src/main/java/com/example/firestore/snippets/model/City.java

```diff
@@ -104,62 +113,43 @@ public void setPopulation(Long population) {
   public void setRegions(List<String> regions) {
     this.regions = regions;
   }
+  // [END_EXCLUDE]
 
-  private String getDefinedValue(String s) {
-    if (s != null) {
-      return s;
-    } else {
-      return "";
-    }
-  }
-
+  // [START_EXCLUDE]
   @Override
   public String toString() {
     StringBuilder sb = new StringBuilder();
     if (name != null) {
-      sb.append(name);
+      sb.append(" name : ").append(name).append(",");
     }
     if (state != null) {
-      sb.append(" state : ");
-      sb.append(state);
-      sb.append(",");
+      sb.append(" state : ").append(state).append(",");
     }
     if (country != null) {
-      sb.append(", ");
-      sb.append(country);
+      sb.append(" country : ").append(country).append(",");
     }
-    sb.append(" : [");
     if (population != null) {
-      sb.append(" population : ");
-      sb.append(population);
-      sb.append(",");
+      sb.append(" population : ").append(population).append(",");
     }
     if (capital != null) {
-      sb.append(" capital : ");
-      sb.append(capital);
-      sb.append(",");
+      sb.append(" capital : ").append(capital).append(",");
     }
     if (regions != null) {
-      sb.append(" regions : [");
-      for (String r : regions) {
-        sb.append(r);
-        sb.append(", ");
-      }
-      sb.append("],");
+      sb.append(" regions : ").append(regions).append(",");
     }
     // remove trailing comma
-    if (sb.lastIndexOf(",") >= sb.length() - 1) {
-      sb.deleteCharAt(sb.length() - 1);
+    if (sb.length() > 0) {
+      sb.setLength(sb.length() - 1);
     }
-    sb.append(" ]");
     return sb.toString();
   }
 
   @Override
   public boolean equals(Object obj) {
-    if (!(obj instanceof City)) {
-      return false;
-    }
+    if (obj == this) return true;
```

Comment:
We need to fix the check style violations:

```
Run mvn -P lint --quiet --batch-mode checkstyle:check
Error:  src/main/java/com/example/firestore/snippets/model/City.java:[21,1] (imports) CustomImportOrder: Wrong lexicographical order for 'com.google.firebase.firestore.DocumentId' import. Should be before 'java.util.Objects'.
Error:  src/main/java/com/example/firestore/snippets/model/City.java:[14[9](https://github.com/googleapis/java-firestore/runs/5292522215?check_suite_focus=true#step:4:9),5] (blocks) NeedBraces: 'if' construct must use '{}'s.
Error:  src/main/java/com/example/firestore/snippets/model/City.java:[[15](https://github.com/googleapis/java-firestore/runs/5292522215?check_suite_focus=true#step:4:15)0,5] (blocks) NeedBraces: 'if' construct must use '{}'s.
Error:  src/main/java/com/example/firestore/snippets/model/City.java:[151,5] (blocks) NeedBraces: 'if' construct must use '{}'s.
Error:  Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.2:check (default-cli) on project firestore-snippets: You have 4 Checkstyle violations. -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error: Process completed with exit code 1.
111

Reply:
Fixed as requested.